### ISSUE TITLE
Building current master fails due to missing files

### ISSUE THREAD

**Issue by eli-osherovich** [tensorflow/tensorflow#50145]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Debian 11
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): source
- TensorFlow version: git/master
- Python version: 3.9
- Installed using virtualenv? pip? conda?: conda
- Bazel version (if compiling from source): 4.1.0
- GCC/Compiler version (if compiling from source): 11
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

**Describe the problem**

Building TF from the current source on GH fails due to missing files (all from `llvm-openmp`):
`tools.pm`
`kmp.h`
`kmp_platform.h`
`kmp_os.h`

**Provide the exact sequence of commands / steps that you executed before running into the problem**
My build command was as follows:
```bash
bazel build  --config=mkl -config=nogcp --config=nonccl  -c opt --copt=-march=native --copt=-O3 -s //tensorflow/tools/pip_package:build_pip_package
```

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by eli-osherovich** [comment]
PR: #50143

**Comment by tilakrayal** [comment]
The issue will move to closed status once the PR is merged.

**Comment by tilakrayal** [comment]
@eli-osherovich ,

Please free feel to move this issue to closed status.Thanks!

**Comment by eli-osherovich** [comment]
@tilakrayal 
Original PR messed up after a rebase.
Re-created it.

**Comment by yimeisun123** [comment]
@eli-osherovich, @tilakrayal, Intel is also seeing this build issue and investigating for solution. As a workaround before any fix, please try with adding this build option,  **--spawn_strategy=standalone** , to the build command and this should temporarily fix the build failure.

**Comment by yimeisun123** [comment]
Tensorflow has reverted a change that causes the build failure, there is no need to use the additional option mentioned above if you sync the project today which has this commit https://github.com/tensorflow/tensorflow/commit/763ae9bed64834d6a9a2e18d9eead0d8763df079 (the commit message is incorrect).

**Comment by eli-osherovich** [comment]
@yimeisun123 
The above change just puts standalone into config. My fix is better - it keeps the usual parallel build

**Comment by yimeisun123** [comment]
@eli-osherovich - tried your earlier PR#50143, and still has build failure. I see that you pushed a new PR#50179, will check. Thanks.

**Comment by eli-osherovich** [comment]
@yimeisun123  it definitely works for me.

By the way, if you are working on TF inside Intel, have a look at this issue: #50176 . There is a wired problem inside Intel's code with GCC 11. While trivial code changes can solve it, the problem is quite interesting. Probably worth solving without code changes.

**Comment by eli-osherovich** [comment]
@tilakrayal  can we move forward with this?
